src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -439,6 +439,37 @@ const mockedOnboardingData: Array<InstitutionOnboardingInfoResource> = [
       },
     },
   },
+  {
+    institution: {
+      id: '92078570527',
+      billingData: {
+        businessName: 'Comune di Bollate AOO',
+        registeredOffice: 'Bollate, Piazza Colonna 370',
+        zipCode: '20021',
+        digitalAddress: 'comune.bollate@pec.it',
+        taxCode: '92078570527',

```

Comment:
this EC taxCode is not onboarded for prod-interop, you should simulate it being onboarded